hydromt_wflow/workflows/river.py

```diff
@@ -32,31 +27,31 @@ def power_law(x, a, b):
 
 def river(
     ds,
-    ds_like=None,
+    ds_model=None,
     river_upa=30.0,
-    slope_len=1e3,
-    channel_dir="up",
+    slope_len=2e3,
     min_rivlen_ratio=0.1,
+    channel_dir="up",
     logger=logger,
-    **kwargs,
 ):
     """Returns river maps
 
     The output maps are:\
     - rivmsk : river mask based on upstream area threshold on upstream area\
     - rivlen : river length [m], minimum set to 1/4 cell res\
     - rivslp : smoothed river slope [m/m]\
-    - rivwth_obs : river width at pixel outlet (if in ds)
-    - rivbed : elevation of the river bed based on pixel outlet 
+    - rivzs : elevation of the river bankfull height based on pixel outlet 
+    - rivwth : river width at pixel outlet (if in ds)
```

Comment:
is it good to mention that this is the bankfull river width here? (it depends of course on the river_geom_fn)

Reply:
It's probably a width close to bank full that is in most datasets, but depending on landsat overpass etc.